src/org/omegat/filters4/xml/openxml/MsOfficeFileFilter.java

```diff
@@ -115,6 +115,11 @@ private void defineDOCUMENTSOptions(Map<String, String> config) {
 
         TRANSLATABLE = Pattern.compile(DOCUMENTS);
     }
+    
+    public MsOfficeFileFilter() {
+        super();
+        defineDOCUMENTSOptions(new java.util.HashMap<String,String>()); // Define the documents to read
```

Comment:
Why don't import HashMap?

Reply:
Usually I work with a basic editor (Kate or Kwrite) so it does not automatically go to top in order to add the imports, as does Eclipse or Netbeans.
The checkstyle says nothing about this, so I did not consider it to be changed.
But I can do, if I find the time.